static/docs/user-guide/autocomplete.md

```diff
@@ -73,6 +73,36 @@ fi
 You can `source` your `~/.bash_profile` or launch a new terminal to utilize
 completion.
 
+<details>
+
+### Click to expand if it doesn't work on Debian/Ubuntu
+
+As mentioned above, it should work out of the box. But if it does not, try these steps:
+
+- Make sure that the package `bash-completion` is installed:
+  ```bash
+  sudo apt install --reinstall bash-completion 
+  ```
+- Make sure that it is enabled. Edit `~/.bashrc` and make sure that these lines are there:
```

Comment:
please, follow the documentation contributing guide to enable the pre-commit hook to indent and wrap things properly for you. And please follow the conventions mentioned there.

Reply:
`it is` -> it's